### ISSUE TITLE
Directus.io/careers/software-engineer/  something wrong with your 404 Not Found page

### ISSUE THREAD

**Issue by MohamedHakem** [directus/directus#10744]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

It's a small issue, but I think it's worth fixing. 

I clicked on this link: 
https://directus.io/careers/software-engineer/

From this tweet:
https://twitter.com/directus/status/1402325136615817219

And I've landed on a 404-Not-Found page, but the CSS of the page is broken: 
![directus-404-page-before](https://user-images.githubusercontent.com/29775958/147624519-9fdeba94-4bc6-4b73-a346-b03d9373a5dd.JPG)
(Before)

It's probably an easy fix, for example, when I remove the (height: 100%) on #missing .hero  from the missing.scss file on line 9 
(below screenshot) the hero section looks much better: 

![directus-404-page-hero-section-fix](https://user-images.githubusercontent.com/29775958/147624956-bf674f23-d41d-41e2-a50d-896577eebe2e.JPG)
(After)

### To Reproduce

Open this link 
https://directus.io/careers/software-engineer/

And you'll find the page's CSS is broken.

### Errors Shown

There's a few: (probably has nothing to do with the CSS of the page)
![logged-errors](https://user-images.githubusercontent.com/29775958/147624630-25153a2e-b742-49d0-85a6-84d4edc170a9.JPG)

### What version of Directus are you using?

I'm not yet using Directus!

### What version of Node.js are you using?

Irrelevant

### What database are you using?

Irrelevant

### What browser are you using?

Chrome, Brave, Firfox, Edge, Vivaldi

### What operating system are you using?

Linux, Windows

### How are you deploying Directus?

Not yet.

**Comment by azrikahar** [comment]
Hi! Thank you very much for bringing this to our attention. Looks like the link was moved, but I'll check with the team whether this was intentional or not. In the meantime, feel free to peruse the careers page via this link instead: https://directus.io/careers/

I was going to transfer this issue to a private repo where we track issues pertaining the website, but I'm not sure whether you'll still be able to see my comment if I do that. Hence I made a copy over there and keep this closed here for visibility 👍

> ### Errors Shown
> There's a few: (probably has nothing to do with the CSS of the page)

Yup those should just be due to your browser or browser extensions blocking the analytics, as (typically) indicated by `ERR_BLOCKED_BY_CLIENT`.
